### PR TITLE
Javascript, indent single line (non-block) else statement properly.

### PR DESCRIPTION
**Pull request by mariusk**

### CODE REVIEW

File: web-mode.el [hunk 1]
```diff
@@ -8353,6 +8353,8 @@ another auto-completion with different ac-sources (e.g. ac-php)")
     (cond
      ((or (null open-ctx) (null (plist-get open-ctx :pos)))
       (setq offset initial-column))
+     ((string-match-p "else$" prev-line)
```
Comment (DoMiNeLa10) [inline web-mode.el]: What about trailing whitespace?

Comment (mariusk) [reply web-mode.el]: Wasn't an issue at my side, probably due to related whitespace removal settings, but I've tried adding support for it anyway (to the best of my elisp regexp abilities, which aren't nearly as good as my perl experience).

**Comment by fxbois** [comment]
@mariusk sorry to be late on this but could you provide an example that triggers the issue you want to fix ?

**Comment by mariusk** [comment]
Sure!
```javascript
let a;
if (true) {
  a = 1;
} else
a = 2;
```
That last line should be indented. Thanks.

**Comment by fxbois** [comment]
@mariusk thx, will be solved with next commit

**Pull request closed without merging**